rebasehelper/constants.py

```diff
@@ -20,8 +20,32 @@
 # Authors: Petr Hracek <phracek@redhat.com>
 #          Tomas Hozza <thozza@redhat.com>
 
+import os
+
 PROGRAM_DESCRIPTION = "Tool to help package maintainers rebase their packages to the latest upstream version"
 NEW_ISSUE_LINK = 'https://github.com/rebase-helper/rebase-helper/issues/new'
 
+REBASE_HELPER_RESULTS_DIR = "rebase-helper-results"
+REBASE_HELPER_WORKSPACE_DIR = "rebase-helper-workspace"
+REBASE_HELPER_REBASED_SOURCES_DIR = os.path.join(REBASE_HELPER_RESULTS_DIR, "rebased-sources")
+
+REBASE_HELPER_LOGS = "logs"
+REBASE_HELPER_DEBUG_LOG = 'debug.log'
+REBASE_HELPER_REPORT_LOG = 'report.log'
+REBASE_HELPER_LOGS_DIR = os.path.join(REBASE_HELPER_RESULTS_DIR, REBASE_HELPER_LOGS)
```

Comment:
This is not used anywhere, but `REBASE_HELPER_LOGS` is. That doesn't make much sense to me, and it's also inconsistent with `REBASE_HELPER_REBASED_SOURCES_DIR` for example.

Reply:
Actually my point was to use `REBASE_HELPER_LOGS_DIR` and not `REBASE_HELPER_LOGS`. The inconsistency remains. Take look at this for example: https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/application.py#L347-L351

Reply:
Or the other way around, drop `REBASE_HELPER_REBASED_SOURCES_DIR`, I'm not sure which is better.

Reply:
It probably makes more sense to use `REBASE_HELPER_LOGS_DIR`. `REBASE_HELPER_LOGS` is always used to create a path but the path is created in the code. It is not used in any other way so it is better to have the path as a constant.